clickhouse_driver/block.py

```diff
@@ -151,17 +152,46 @@ def get_column_by_index(self, index):
         return [row[index] for row in self.data]
 
     def _mutate_dicts_to_rows(self, data):
-        column_names = [x[0] for x in self.columns_with_types]
-
         check_row_type = False
         if self.types_check:
             check_row_type = self._check_dict_row_type
 
+        return self._pure_mutate_dicts_to_rows(
```

Comment:
These changes in `_mutate_dicts_to_rows` can slow down INSERTs. Do you have any estimations for possible slowing down? Let's see on some 'simple' example without nesting of `Nested` types and significant number of rows. 

May be fall back to `[row[name] for name in column_names]` if there is no columns for extra transformations? This is a bit ugly, but can save time during INSERTs.

Do we need to change `ColumnOrientedBlock` as well?

Reply:
I can add a if/else in line 177 if `all(x[1] is None for x in columns_with_cwt)` and do simple insert

I don't think we need to change `ColumnOrientedBlock` because we're not passing list of dict to it.

Reply:
Nevermind.

I've tested new code with following snippet

```python
from clickhouse_driver import Client

cols = [
    'Year', 'Quarter', 'Month', 'DayofMonth', 'DayOfWeek', 'AirlineID', 'OriginAirportID', 'OriginAirportSeqID',
    'OriginCityMarketID', 'OriginWac', 'DestAirportID', 'DestAirportSeqID', 'DestCityMarketID', 'DestWac',
    'CRSDepTime', 'DepTime', 'DepDelay', 'DepDelayMinutes', 'DepDel15', 'TaxiOut', 'WheelsOff', 'WheelsOn',
    'TaxiIn', 'CRSArrTime', 'ArrTime', 'ArrDelay', 'ArrDelayMinutes', 'ArrDel15', 'ArrivalDelayGroups',
    'Cancelled', 'Diverted', 'CRSElapsedTime', 'ActualElapsedTime', 'AirTime', 'Flights', 'Distance',
    'DistanceGroup', 'CarrierDelay', 'WeatherDelay', 'NASDelay', 'SecurityDelay', 'LateAircraftDelay',
    'Div1AirportID', 'Div1AirportSeqID', 'Div2AirportID', 'Div2AirportSeqID', 'Div3AirportID',
    'Div3AirportSeqID', 'Div4AirportID', 'Div4AirportSeqID', 'Div5AirportID', 'Div5AirportSeqID'
]

query = "SELECT {} FROM perftest.ontime WHERE FlightDate < '2017-02-18'".format(', '.join(cols))
client = Client.from_url('clickhouse://localhost')
rv = client.execute(query)
rv = [dict(zip(cols, x)) for x in rv]

%time client.execute('INSERT INTO perftest.test ({}) VALUES'.format(', '.join(cols)), rv)
```

Time is the same as master branch.